index.html

```diff
@@ -1,40 +1,180 @@
-    <title>Mobile version Replica</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="./style.css">
-
-
-</head>
-<body>
-    <div class="container">
-        <header>
-            <nav class="navbar">
-                <li class="item"><a href=""><img src="/icons/bar.png" alt=""></a></li>
-                <li class="item"><a href=""><img src="/icons/bar.png" alt=""></a></li>
-                <li class="item"><a href=""><img src="/icons/bar.png" alt=""></a></li>
-            </nav>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+        />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" href="style.css" />
+        <title>Portfolio</title>
+    </head>
+    <body>
+        <header class="bkg">
+            <button class="menu">
+                <span class="material-symbols-outlined"> menu </span>
+            </button>
         </header>
+        <!-- Headline -->
+        <section class="spacing headline">
+            <h1 class="title">Charlie Giorgania</h1>
+            <p class="paragraph-headline">
+                Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+                works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+            <a class="link-headline" href="">Start collaboration</a>
+        </section>
+        <main>
+            <!-- Projects -->
+            <section>
+                <h2 class="title spacing">Projects</h2>
+                <div class="projects">
+                    <div>
+                        <img src="/icons/pc1.png" alt="" />
+                        <div class="card-content">
+                            <h3 class="subtitle">Project name goes here</h3>
+                            <ul class="technologies">
+                                <li>HTML/CSS</li>
+                                <li>Ruby on Rails</li>
+                                <li>Javascript</li>
+                            </ul>
+                            <button class="button">
```

Comment:
- [ ] Please add hover and pressed state to all your buttons accordingly. You can use the `:hover` and `:active` pseudo-classes.
![btn5](https://user-images.githubusercontent.com/94551484/206829591-a2f23173-f045-46ec-9baf-dffa81c5bfd4.png)